src/test/java/com/linecorp/armeria/client/http/SimpleHttpClientIntegrationTest.java

```diff
@@ -48,15 +47,11 @@
     private static RemoteInvokerFactory remoteInvokerFactory;
 
     static {
-        final SelfSignedCertificate ssc;
         final ServerBuilder sb = new ServerBuilder();
 
         try {
             sb.port(0, SessionProtocol.HTTP);
 
-            ssc = new SelfSignedCertificate("127.0.0.1");
-            sb.sslContext(SessionProtocol.HTTPS, ssc.certificate(), ssc.privateKey());
-
```

Comment:
Just c​uriosity. Why is this code removed? Because `SimpleHttpClientSniTest` covers this?